src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -4,6 +4,7 @@
 #include "Replayer.h"
 #include "common/debug.h"
 #include "common/errno.h"
+#include "common/perf_counters_key.h"
```

Comment:
Include `common/perf_counters.h` before this line just like for the journal replayer.